nexus/src/app/mod.rs

```diff
@@ -79,8 +79,10 @@ pub(crate) use nexus_db_queries::db::queries::disk::MAX_DISKS_PER_INSTANCE;
 
 pub(crate) const MAX_NICS_PER_INSTANCE: usize = 8;
 
-// TODO-completeness: Support multiple external IPs
-pub(crate) const MAX_EXTERNAL_IPS_PER_INSTANCE: usize = 1;
+// XXX: Might want to recast as max *floating* IPs, we have at most one
+//      ephemeral (so bounded in saga by design).
+pub(crate) const MAX_EXTERNAL_IPS_PER_INSTANCE: usize = 32;
```

Comment:
Where does the upper bound of 32 come from?

Reply:
Ok. This additional color should probably be captured in a comment here in case we need to come back and revisit this limit. We'll also want to note this limit in the user-facing docs.